website/docs/configuration/data-sources.html.md

```diff
@@ -125,7 +125,7 @@ resources.
 
 ## Multiple Resource Instances
 
-Data resources support [the `count` meta-argument](/docs/configuration/resources.html#multiple-resource-instances)
+Data resources support [the `count` meta-argument](./resources.html#multiple-resource-instances)
```

Comment:
I think this fragment will now be `#count-multiple-resource-instances` after the changes to headings on the resources page. Perhaps this applies to the other meta-argument links on this page too.

Reply:
Blaugh, I think I fixed it once and then unfixed it during an interactive rebase. Let's try this again!

Reply:
Oh, nope, I actually only fixed half of them originally 😅 Thanks for the catch!